src/components/sbb-breadcrumb-group/sbb-breadcrumb-group.e2e.ts

```diff
@@ -1,48 +1,52 @@
-import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
 import { waitForCondition } from '../../global/testing';
+import { assert, expect, fixture } from '@open-wc/testing';
+import { html } from 'lit/static-html.js';
+import { sendKeys, setViewport } from '@web/test-runner-commands';
+import { EventSpy, waitForLitRender } from '../../global/testing';
+import { SbbBreadcrumbGroup } from './sbb-breadcrumb-group';
+import { SbbBreadcrumb } from '../sbb-breadcrumb';
+import '../sbb-breadcrumb';
 
 describe('sbb-breadcrumb-group', () => {
   describe('without ellipsis', () => {
-    let element: E2EElement, page: E2EPage;
+    let element: SbbBreadcrumbGroup;
 
     beforeEach(async () => {
-      page = await newE2EPage();
-      await page.setContent(`
+      element = await fixture(html`
         <sbb-breadcrumb-group>
           <sbb-breadcrumb href="/" icon-name="house-small" id="breadcrumb-0"></sbb-breadcrumb>
           <sbb-breadcrumb href="/" id="breadcrumb-1">One</sbb-breadcrumb>
           <sbb-breadcrumb href="/" id="breadcrumb-2">Two</sbb-breadcrumb>
```

Comment:
Are you sure that `href=/` does not create a problem in the test environment? (In case, try `href=#`)

Reply:
I changed all of them, thanks